### PR TITLE
Feature/filter practices country

### PR DESCRIPTION
**Pull request by Bluesmile82**
Filter the practices by country and land use type.
https://vizzuality.atlassian.net/browse/ORC-371?atlOrigin=eyJpIjoiNDAyZWUzNzkwNjk3NDk0ZjlkZTM5Njk5MzI0NjI3YWYiLCJwIjoiaiJ9

### CODE REVIEW

File: client/src/app/(modules)/practices/(main)/filters-sidebar.tsx [hunk 1]
```diff
@@ -56,7 +60,40 @@ export default function FiltersSidebar() {
...
         </Button>
         <h1 className="font-serif text-3.8xl">Filters</h1>
-        <div className="flex flex-col gap-y-10">Coming soon</div>
+        <div className="flex flex-col gap-y-10">
+          <fieldset className="relative">
+            <Button
+              type="button"
+              variant="vanilla"
+              size="auto"
+              className="absolute bottom-full right-0 -translate-y-4 text-base font-semibold text-blue-500 hover:text-blue-800 disabled:text-gray-300 disabled:opacity-100"
```
Comment (clementprdhomme) [inline client/src/app/(modules)/practices/(main)/filters-sidebar.tsx:69]: This button should be brown, not blue.

File: client/src/app/(modules)/practices/(main)/filters-sidebar.tsx [hunk 2]
```diff
@@ -56,7 +60,40 @@ export default function FiltersSidebar() {
...
+              onClick={() =>
+                setFilters({
```
Comment (clementprdhomme) [inline client/src/app/(modules)/practices/(main)/filters-sidebar.tsx:71]: You forgot to reset the land use type.

File: client/src/app/(modules)/practices/(main)/filters-sidebar.tsx [hunk 3]
```diff
@@ -56,7 +60,40 @@ export default function FiltersSidebar() {
...
+                  country: [],
+                })
+              }
+            >
+              Reset all
+            </Button>
+            <div className="space-y-4">
+              <MultiCombobox
+                name="Country"
+                variant="network-organization"
```
Comment (clementprdhomme) [inline client/src/app/(modules)/practices/(main)/filters-sidebar.tsx:82]: Use a new variant to display the filter with a brown colour.

File: client/src/app/(modules)/practices/(main)/filters-sidebar.tsx [hunk 4]
```diff
@@ -56,7 +60,40 @@ export default function FiltersSidebar() {
...
+                value={filters.country ?? []}
+                options={practicesFiltersOptions.country}
+                onChange={(value) => setFilters({ ...filters, country: value as number[] })}
+              />
+              <MultiCombobox
+                name="Land use types"
+                variant="network-organization"
```
Comment (clementprdhomme) [inline client/src/app/(modules)/practices/(main)/filters-sidebar.tsx:89]: Use a new variant to display the filter with a brown colour.

File: client/src/app/(modules)/practices/(main)/filters-sidebar.tsx [hunk 5]
```diff
@@ -56,7 +60,40 @@ export default function FiltersSidebar() {
...
+            <div className="space-y-4">
+              <MultiCombobox
+                name="Country"
+                variant="network-organization"
+                value={filters.country ?? []}
+                options={practicesFiltersOptions.country}
+                onChange={(value) => setFilters({ ...filters, country: value as number[] })}
+              />
+              <MultiCombobox
+                name="Land use types"
```
Comment (clementprdhomme) [inline client/src/app/(modules)/practices/(main)/filters-sidebar.tsx:88]:
```suggestion
                name="Land use type"
```

File: client/src/app/(modules)/practices/(main)/filters-sidebar.tsx [hunk 6]
```diff
@@ -56,7 +60,40 @@ export default function FiltersSidebar() {
...
+              onClick={() =>
+                setFilters({
+                  ...filters,
+                  country: [],
+                })
+              }
+            >
+              Reset all
+            </Button>
+            <div className="space-y-4">
```
Comment (clementprdhomme) [inline client/src/app/(modules)/practices/(main)/filters-sidebar.tsx:79]: I think that the land use type filter should be first 😉.

**Review by clementprdhomme — COMMENTED** [review]
In addition to my inline comments, I think there might be an issue with the data or filters.

To test it, zoom on France. By default you will see 14 practices. Now, filter by land use type but select all the possible options. Once you do this, there are only 12 practices remaining. This number should still be 14.

Can you check what's happening?

**Comment by Bluesmile82** [comment]
On the data there are some practices without any associated Land use type that is why the values are missing for France. I think science is aware of this. Anyway I will remind it on the channel

**Review by clementprdhomme — APPROVED** [review]

**Pull request merged**